package.json

```diff
@@ -130,5 +130,15 @@
     "vinyl-buffer": "^1.0.1",
     "vinyl-source-stream": "^2.0.0",
     "yargs": "^16.0.3"
-  }
+  },
+  "files": [
+    "/dist/fonts",
+    "/dist/img",
+    "/dist/js",
+    "/dist/scss",
+    "/src/js",
+    "/build/components/render",
+    "CONTRIBUTING.md",
+    "SECURITY.md"
```

Comment:
Note that the following are included by default:
- `README.md`
- `LICENSE.md`
- `package.json`

Reply:
Why not just `/dist` is written out for clarity?